modules/op-configuring-pipelines-as-code-for-a-github-app.adoc

```diff
@@ -70,20 +68,22 @@ For {pac} installated by default using the {pipelines-title} Operator, use the `
 
 . In the **Private keys** section, click **Generate Private key** to automatically generate and download a private key for the GitHub app. Securely store the private key for future reference and usage.
 
+. Install the created App on a repository that you want to use with Pipelines as Code.
+
 
 [id="configuring-pac-for-github-app"]
 == Configuring {pac} to access a GitHub App
 
 To configure {pac} to access the newly created GitHub App, execute the following command:
-+
+
 [source,terminal]
 ----
 $ oc -n <pipelines-as-code> create secret generic pipelines-as-code-secret \ <1>
         --from-literal github-private-key="$(cat <PATH_PRIVATE_KEY>)" \ <2>
         --from-literal github-application-id="<APP_ID>" \ <3>
         --from-literal webhook.secret="<WEBHOOK_SECRET>" <4>
 ----
-<1> For {pac} installated by default using the {pipelines-title} Operator, use the `openshift-pipelines` namespace instead of `pipelines-as-code`.
+<1> For {pac} installated by default using the {pipelines-title} Operator, use the `pipelines-as-code` namespace instead of `openshift-pipelines`.
```

Comment:
If pac is installed directly without using the Operator then use `pipelines-as-code` instead of `openshift-pipelines`